src/main/java/commands/ListCommand.java

```diff
@@ -96,10 +99,15 @@ private String listChapters(Access access) {
         }
         result.append(String.format(MESSAGE_EXIST, CHAPTER));
         for (Chapter c : allChapters) {
+            if (c.getChapterName().equals("dues")) {
```

Comment:
instead of checking for dues here, you can check in loadChapter in storage instead. so that it wont add the due folder as a chapter object